lms/migrations/versions/3d0c022c716c_add_missing_indexes_for_fks.py

```diff
@@ -0,0 +1,57 @@
+"""Add missing indexes for FKs."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "3d0c022c716c"
+down_revision = "7858a12c4e4a"
+
+
+def upgrade() -> None:
+    # CONCURRENTLY can't be used inside a transaction. Finish the current one.
+    op.execute("COMMIT")
+
+    op.create_index(
+        op.f("ix__assignment_grouping_assignment_id"),
+        "assignment_grouping",
+        ["assignment_id"],
+        unique=False,
+        postgresql_concurrently=True,
+    )
+    op.create_index(
+        op.f("ix__assignment_grouping_grouping_id"),
+        "assignment_grouping",
+        ["grouping_id"],
+        unique=False,
+        postgresql_concurrently=True,
+    )
+    op.create_index(
+        op.f("ix__assignment_membership_assignment_id"),
+        "assignment_membership",
+        ["assignment_id"],
+        unique=False,
+        postgresql_concurrently=True,
+    )
+    op.create_index(
+        op.f("ix__grouping_membership_user_id"),
+        "grouping_membership",
+        ["user_id"],
+        unique=False,
+        postgresql_concurrently=True,
+    )
+
+
+def downgrade() -> None:
+    op.drop_index(
+        op.f("ix__grouping_membership_user_id"), table_name="grouping_membership"
+    )
```

Comment:
I notice that we do concurrent creation of indexes but not deletion. I don't think we've ever had to do a downgrade in production, but if we did this might be relevant - https://squawkhq.com/docs/require-concurrent-index-deletion/.

Reply:
We'd never just downgrade a migration on postgres but create a new migration to achieve the same and keep.

But to write that migration it would be nice that we can just copy from the failed migration's downgrade, added the concurrently option there :+1: